app/components/DataCard/index.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+
+export const Div = styled.div`
```

Comment:
Be specific & descriptive in naming styled components. use descriptive names as you would do in naming css classes

e.g. instead of Div, you can name it DataCardDiv or StyledDiv